src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -69,6 +88,23 @@ case class AgoraEntity(@(ApiModelProperty@field)(required = true, value = "The n
                        payload: Option[String] = None
                         )
 
+@ApiModel(value = "Agora Search")
```

Comment:
1) AgoraSearch doesn't have a createDate field, which helps me avoid complications in MethodsService.
2) Later we might want to have beforeDate and afterDate fields so people can specify a date range. But AgoraEntity has just one date field.

I did this mainly because of 1)